applications/FemToDemApplication/custom_constitutive/elastic_isotropic_3d.h

```diff
@@ -0,0 +1,388 @@
+// KRATOS  ___|  |                   |                   |
```

Comment:
You now how to avoid this copy-paste...I assume this is a transition

Reply:
sure, as soon as I depend on structural mech app this will be erased